appveyor.yml

```diff
@@ -32,94 +22,187 @@ skip_commits:
 #    environment configuration    #
 #---------------------------------#
 
-# Operating system (build VM template)
-os: Visual Studio 2015 CTP
+# http://www.appveyor.com/docs/environment-variables
+# http://www.appveyor.com/docs/installed-software
+
+# Build VM template
+#os: Windows Server 2012
```

Comment:
Windows Server 2012 R2 (x64) - We were using another VM & OS for Visual Studio before it's release (CTP candidates), but can now revert to this default.